source/dub/dub.d

```diff
@@ -1732,9 +1733,9 @@ private class DependencyVersionResolver : DependencyResolver!(Dependency, Depend
 			if (rootpack == name) {
 				try {
 					auto desc = ps.fetchPackageRecipe(name, dep, prerelease);
-					if (desc.type == Json.Type.null_)
+					if (desc.isNull)
 						continue;
-					auto ret = new Package(desc);
+					auto ret = new Package(desc.get);
```

Comment:
I think that was also the last use of the `Package` constructor that takes a JSON, so feel free to deprecate it.